spec/timezone.html

```diff
@@ -464,10 +472,21 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-formattimezoneoffsetstring" aoid="FormatTimeZoneOffsetString">
-      <h1>FormatTimeZoneOffsetString ( _offsetNanoseconds_ )</h1>
+    <emu-clause id="sec-temporal-formattimezoneoffsetstring" type="abstract operation">
+      <h1>
+        FormatTimeZoneOffsetString (
+          _offsetNanoseconds_: an integer
+        ): a String
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It formats a time zone offset, in nanoseconds, into a UTC offset string.
+          If _offsetNanoseconds_ represents an integer number of minutes, then the output will be in ±HH:MM format.
+          Otherwise, the output will be in ±HH:MM:SS[.S[S[S[S[S[S[S[S[S]]]]]]]]] format.
```

Comment:
```suggestion
          If _offsetNanoseconds_ represents an integer number of minutes, then the output will be formatted like ±HH:MM.
          Otherwise, the output will be formatted like ±HH:MM:SS or (if the offset does not evenly divide into seconds) ±HH:MM:SS.fff… where the "fff" part is a sequence of at least 1 and at most 9 fractional seconds digits with no trailing zeroes.
```